utam-core/src/main/java/utam/core/driver/DriverConfig.java

```diff
@@ -40,28 +40,35 @@
   private final Duration implicitTimeout;
   private final Duration explicitTimeout;
   private final Duration pollingInterval;
+  private final String bridgeAppTitle;
 
   /**
    * Initializes a new instance of the DriverConfig class
    *
    * @param implicitTimeout the implicit wait timeout for finding elements
    * @param explicitTimeout the explicit wait timeout for actions
    * @param pollingInterval the polling interval used to poll for conditions
+   * @param bridgeAppTitle  bridge app title for mobile app
    */
   public DriverConfig(
       Duration implicitTimeout,
       Duration explicitTimeout,
-      Duration pollingInterval) {
+      Duration pollingInterval,
+      String bridgeAppTitle) {
```

Comment:
Will this break things at runtime because the constructor signature changes? If not, that's okay, but we really don't want to break runtime of POs compiled with earlier compiler versions.